static/lato.css

```diff
@@ -0,0 +1,93 @@
+/* Adapted from https://fonts.googleapis.com/css?family=Lato:400,400i,700,900 */
+
+/* non-latin (cyrillic, hungarian, serbian) */
+@font-face {
+    font-family: 'Lato';
+    font-style: italic;
+    font-weight: 400;
+    src: local('Lato Italic'), local('Lato-Italic'), url(fonts/lato/Lato-Italic.woff2) format('woff2');
+    unicode-range: U+0400-04FF, U+0500-052F, U+2DE0-2DFF, U+A640-A69F, U+1D00-1D7F;
+}
+
+@font-face {
+    font-family: 'Lato';
+    font-style: normal;
+    font-weight: 400;
+    src: local('Lato Regular'), local('Lato-Regular'), url(fonts/lato/Lato-Regular.woff2) format('woff2');
+    unicode-range: U+0400-04FF, U+0500-052F, U+2DE0-2DFF, U+A640-A69F, U+1D00-1D7F;
+}
+
+@font-face {
+    font-family: 'Lato';
+    font-style: normal;
+    font-weight: 700;
+    src: local('Lato Bold'), local('Lato-Bold'), url(fonts/lato/Lato-Bold.woff2) format('woff2');
+    unicode-range: U+0400-04FF, U+0500-052F, U+2DE0-2DFF, U+A640-A69F, U+1D00-1D7F;
+}
+
+@font-face {
+    font-family: 'Lato';
+    font-style: normal;
+    font-weight: 900;
+    src: local('Lato Black'), local('Lato-Black'), url(fonts/lato/Lato-Black.woff2) format('woff2');
+    unicode-range: U+0400-04FF, U+0500-052F, U+2DE0-2DFF, U+A640-A69F, U+1D00-1D7F;
+}
+
+/* latin */
+@font-face {
+    font-family: 'Lato';
+    font-style: italic;
+    font-weight: 400;
+    src: local('Lato Italic'), local('Lato-Italic'), url(fonts/lato/LatoLatin-Italic.woff2) format('woff2');
+}
+
+@font-face {
+    font-family: 'Lato';
+    font-style: normal;
+    font-weight: 400;
+    src: local('Lato Regular'), local('Lato-Regular'), url(fonts/lato/LatoLatin-Regular.woff2) format('woff2');
+}
+
+@font-face {
+    font-family: 'Lato';
+    font-style: normal;
+    font-weight: 700;
+    src: local('Lato Bold'), local('Lato-Bold'), url(fonts/lato/LatoLatin-Bold.woff2) format('woff2');
+}
+
+@font-face {
+    font-family: 'Lato';
+    font-style: normal;
+    font-weight: 900;
+    src: local('Lato Black'), local('Lato-Black'), url(fonts/lato/LatoLatin-Black.woff2) format('woff2');
+}
+
+/* Latin Extended (e.g. vietnamese) */
+@font-face {
+    font-family: 'Lato';
+    font-style: italic;
+    font-weight: 400;
+    src: local('Lato Italic'), local('Lato-Italic'), url(fonts/lato/LatoLatinExtended-Italic.woff2) format('woff2');
+    unicode-range: U+0020-002F, U+0030-0039, U+003A-0040, U+0041-005A, U+005B-0060, U+0061-007A, U+007B-007E, U+00C0-00C3, U+00C8-00CA, U+00CC-00CD, U+00D0, U+00D2-00D5, U+00D9-00DA, U+00DD, U+00E0-00E3, U+00E8-00EA, U+00EC-00ED, U+00F2-00F5, U+00F9-00FA, U+00FD, U+0102-0103, U+0110-0111, U+0128-0129, U+0168-0169, U+01A0-01B0, U+1EA0-1EF9, U+02C6-0323;
+}
```

Comment:
Some of the ranges overlap what the non-extended LatoLatin fonts provide.  Are the glyphs different or are we trying to avoid both LatoLatinExtended and LatoLatin on Vietnamese pages?

Reply:
Agree with your observation... I'm running some tests on webapp and will change the ranges based on my findings :)